src/values/Paper.js

```diff
@@ -28,6 +30,10 @@ var Paper = function( text, attributes ) {
 	if ( attributes.locale === "" ) {
 		attributes.locale = defaultAttributes.locale;
 	}
+	const keywordIsNotEmpty = replaceDiacritics( attributes.keyword ).match( /[A-Za-zА-Яа-я0-9]/gi );
```

Comment:
Does that character-range also include arabic, japanese or other asian languages?